src/plugins/intel_cpu/src/cpu_map_scheduling.cpp

```diff
@@ -69,4 +69,57 @@ std::vector<std::vector<int>> apply_hyper_threading(bool& input_ht_hint,
     return result_table;
 }
 
+bool check_cpu_pinning(bool cpu_pinning) {
```

Comment:
why we need to have any argument for this function? Like what is the point to call `check_cpu_pinning(false)` given that it always return `false`?
I would expect to see smt like that in the code:
```
if (cpu_pinning && cpu_pinning_available()) {
    enableCpuPinning = true;
} else {
    enableCpuPinning = false;
}
changedCpuPinning = true;